test/Dialect/FIRRTL/lower-types.mlir

```diff
@@ -1045,18 +1045,23 @@ firrtl.module private @is1436_FOO() {
 
 firrtl.circuit "NLALowering" {
   // Check if the NLA is updated with the new lowered symbol on a field element.
-  firrtl.hierpath @nla [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b]
-  // CHECK: firrtl.hierpath @nla_0 [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b_data]
-  // CHECK: firrtl.hierpath @nla [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b_ready]
-  firrtl.hierpath @nla_1 [@fallBackName::@test, @Aardvark::@test_1, @NLALowering]
-  firrtl.hierpath @nla_2 [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b2]
+  firrtl.hierpath @nla_b [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b]
+  // CHECK:      firrtl.hierpath @nla_b [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b_valid]
+  // CHECK-NEXT: firrtl.hierpath @nla_b_0 [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b_ready]
+  // CHECK-NEXT: firrtl.hierpath @nla_b_1 [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b_data]
+  firrtl.hierpath @nla [@fallBackName::@test, @Aardvark::@test_1, @NLALowering]
+  // CHECK-NEXT: firrtl.hierpath @nla [@fallBackName::@test, @Aardvark::@test_1, @NLALowering]
+  firrtl.hierpath @nla_b2 [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b2]
+  // CHECK-NEXT: firrtl.hierpath @nla_b2 [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b2_valid]
+  // CHECK-NEXT: firrtl.hierpath @nla_b2_0 [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b2_ready]
+  // CHECK-NEXT: firrtl.hierpath @nla_b2_1 [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b2_data]
   // CHECK-NOT: firrtl.hierpath @nla_2
   firrtl.module private @fallBackName() {
     firrtl.instance test sym @test {
       annotations = [
+        {circt.nonlocal = @nla_b, class = "circt.nonlocal"},
```

Comment:
We should get rid of the breadcrumbs !

Reply:
This is fair. I had not changed it to prevent churn in the test, but this test is changed sufficiently that I think it's okay.